src/openms/source/ANALYSIS/XLMS/OPXLHelper.cpp

```diff
@@ -701,33 +702,42 @@ namespace OpenMS
         {
           seq_alpha.setModification(alpha_pos, mods[0]);
           mod_set = true;
+          ph_alpha.setMetaValue("xl_mod", mods[0]);
+          // ph_alpha.setMetaValue("xl_mass", DataValue(top_csms_spectrum[i].cross_link.cross_linker_mass));
```

Comment:
if this is valuable debug code consider adding something like: #ifdef DEBUG_OPXL ...
to enable it